src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRent = 40;
+  const secondDaylimit = 7;
+  const firstDiscount = 50;
+  const secondDiscount = 20;
+  const firstDaylimit = 3;
```

Comment:
First and second describe nothing. Try to give more clear names (for example, basicDiscount, smallDiscountDays)